packages/ui-components/src/components/UIInput/UITextInput.tsx

```diff
@@ -85,8 +85,7 @@ export class UITextInput extends React.Component<UITextInputProps> {
         return {
             ...{
                 root: {
-                    height: 'auto',
-                    fontFamily: 'var(--vscode-font-family)'
```

Comment:
as inputs don't inherit font from the parent, should we leave at least `font: inherit` here?